.github/workflows/ci.yml

```diff
@@ -33,3 +33,20 @@ jobs:
       run: yarn
     - name: pre fomantic install & gulp build
       run: yarn gulp install
+  codeql:
+    name: CodeQL
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+    steps:
+    - uses: actions/checkout@v3
+    - uses: github/codeql-action/init@v2
+      with:
+        languages: 'javascript'
+        config-file: ./.github/codeql/codeql-config.yml
+    - name: Autobuild
+      uses: github/codeql-action/autobuild@v2
+    - name: Perform CodeQL Analysis
+      uses: github/codeql-action/analyze@v2
```

Comment:
The settings from original ci is missing:
```yml
fail-fast: false
      matrix:
        language: [ 'javascript' ]
```

Reply:
but fail-fast is misssing...

Reply:
without a matrix, for one job only, it is not needed

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast and it is a per (matrixed) job config, not per worflow config

tested in https://github.com/fomantic/Fomantic-UI/pull/2611/commits/19a07d17df713013f39711b6d9e8a359fd0fa50c - `Lint` job (also /wo `fail-fast: false`) failed, but the `CodeQL` continued to run (ie. was not cancelled, even if it took much longer)

Reply:
yes, because there is fail-fast in the test block, but if it would fail inside codeql, i want the whole codeql to continue checking for all possible errors. I beieven without fail-fast (inside the codeql block) it will stop parsing after finding the first error, isnt it?

Reply:
I would say no, as in `lint` job there is no `fail-fast: false`. In `test` `fail-fast: false` is present, but that should be not relevant, or is it?

Reply:
🤔 Seems you are right according to the gh docs you posted above (fail-fast only affects multiple matrix entries). Alright, lets keep it then as you did 😄